constellations/gemini-user/src/announcements/announce.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+require("./announce.sass");
+import React from "react";
+import API from "../utils/api.js";
+import { keys, sortBy, groupBy } from "lodash";
+import moment from "moment";
+
+export class AnnouncementGroup extends React.Component {
+    render() {
+        let a = this.props.announcements;
+        var postedAt = moment(this.props.postedAt).format('l');
+        let announcements = a.map((announcement, index) => (
```

Comment:
You define a variable `a` but only use it once.
You might as well get rid of the variable `a` and simple use `this.props.announcements.map(...)`

Reply:
Also because `announcements` here never changes, you can make it a `const`.